src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -185,7 +264,7 @@ public boolean isUpdatedByEtagToRetry(
         return true;
     }
 
-    public boolean isUpdatedByResponseHeadersToRetry(
+    public boolean checkToRetry(
```

Comment:
What does `checkToRetry` mean?

Reply:
it should be actually `checked response header  to retry request`